clang/include/clang/Support/Compiler.h

```diff
@@ -14,8 +14,6 @@
 #ifndef CLANG_SUPPORT_COMPILER_H
 #define CLANG_SUPPORT_COMPILER_H
 
-#include "llvm/Support/Compiler.h"
```

Comment:
The include of this file needs to move to Attr.h, module imports can't be in a namespace, so putting the include in Attrs.inc is wrong.

You should also add a new top level module for the Support directory.

`module Clang_Support { requires cplusplus umbrella "clang/Support" module * { export * } }`

Reply:
Thanks! Fixed!